utilities/src/main/java/io/syndesis/qe/bdd/validation/MqttValidationSteps.java

```diff
@@ -52,9 +53,27 @@ public void sendAndRecieveMessage(String senderTopic, String receiverTopic) {
         }
 
         //check flag
-        Assertions.assertThat(Receiver.RECEIVED_FLAG).as("Message was not received!!!")
+        assertThat(Receiver.RECEIVED_FLAG).as("Message was not received!!!")
                 .isEqualTo(1);
+    }
+
+    @When("^send mqtt message to \"([^\"]*)\" topic$")
+    public void sendMqttMessage(String senderTopic) {
 
+        try {
+            portForward();
+            // give port forward some time
+            TestUtils.sleepIgnoreInterrupt(2000);
```

Comment:
is this really needed?

Reply:
Yes, moved to appropriate place. It failed a few times, better +2sec than unstable tests.